UI/forms/OBSMissingFiles.ui

```diff
@@ -21,7 +21,7 @@
        <property name="minimumSize">
```

Comment:
Could this property be removed altogether?

Reply:
Per off-thread discussion, this can be addressed at a later time.